src/components/usa-section/usa-section.stories.js

```diff
@@ -0,0 +1,28 @@
+import "../../stylesheets/uswds.scss";
+import Component from "./usa-section.twig";
+import Data from "./usa-section.json";
```

Comment:
When there are multiple JSON files, can we standardize the file structure inside a content folder like the button and search components?

Reply:
@amyleadem do you recommend I do this in phase 1 and pull it into phase 2? or just do it in phase 2 to ensure all components match